tests/RobotFramework/README.md

```diff
@@ -110,6 +110,22 @@ Checkout the [dev container instructions](../../docs/src/developer/DEV_CONTAINER
 
 3. Follow the console instructions and edit the `.env` file which was created by the `./bin/setup.sh` script
 
+    It is recommended to provide a local certificate authority (CA) signing certificate so that the system tests can utilize it to generate the device certificates. To use your own signing certificate you need to add the following keys which contain the base64 encoded contents of the CA key and public certificate.
+
+    For example, assuming you have the `~/tedge-ca.key` (private key) and the `~/tedge-ca.crt` public certificate, you can run the following on the console:
+
+    ```sh
+    cat ~/tedge-ca.key | base64
+    cat ~/tedge-ca.crt | base64
+    ```
+
+    Copy the output form above for the respective values in the following environment variables in the `.env` file (which will be used by the system tests).
+
+    ```sh
+    CA_KEY="{base64_encoded_tedge-ca.key}"
+    CA_PUB="{base64_encoded_tedge-ca.crt}"
+    ```
```

Comment:
I would also add a note telling that `~/tedge-ca.crt` has to be uploaded to / trusted by c8y.